### ISSUE TITLE
ADE20k results reproduction

### ISSUE THREAD

**Issue by rayleizhu** [Haoqing-Wang/LocalMIM#3]
I cannot reproduce ADE20k performance (49.5 mIoU) shown in Table 2 of the paper. 

Using your released checkpoint and semantic segmentation code, I got 48.65 mIoU (see the log in [onedrive](https://1drv.ms/u/s!AkBbczdRlZvCrXjadd_kHIlvAmJj?e=pOJSad)).

Any suggestions for the reproduction?

**Comment by Haoqing-Wang** [comment]
I suggest to adjust the learning rate lr from [1e-4, 2e-4, 3e-4, 4e-4, 5e-4], and the layer_decay_rate can also be adjusted. Please check different learning rate first.
https://github.com/Haoqing-Wang/LocalMIM/blob/314a6e558a8c47bf0ef146235e2ced8d01e27fe6/semantic_segmentation/configs/upernet_vit_base_12_512_slide_160k_ade20k.py#L23

**Comment by rayleizhu** [comment]
The current config is consistent with the description in your paper (p13). 

![image](https://github.com/Haoqing-Wang/LocalMIM/assets/19933627/730d552c-dc1f-4b20-a7cd-23ef1f1514ef)

1. Can you release the training log? If not,
2. Can you release the config you used to produce the result in your paper? If not,
3. Can you recall the critical hyperparameters you actually used?
